crates/matrix-sdk-common/src/lib.rs

```diff
@@ -1,9 +1,7 @@
 #![doc = include_str!("../README.md")]
 #![warn(missing_debug_implementations)]
 
-pub use async_trait::async_trait;
 pub use instant;
```

Comment:
do we still need `instant` now?

Reply:
We don't, but the dependency specification for it currently is spread across two `cfg`s, one with and one without the `js` feature. If / when we remove that, I think we can remove the re-export as well.